flutter_web_auth_2/example/pubspec.yaml

```diff
@@ -8,9 +8,11 @@ environment:
 
 dependencies:
   cupertino_icons: ^1.0.3
+  desktop_webview_window: ^0.2.3
   flutter:
     sdk: flutter
-  flutter_web_auth_2: ^3.0.0
+  flutter_web_auth_2:
+    path: ../../flutter_web_auth_2
```

Comment:
Melos should do that automatically when calling `bootstrap`, but using a cleaner `pubspec_override`

Reply:
Sorry I'm not familiar with melos and this was faster for me. I can revert this to the original if you'd like

Reply:
No problem! :) And thanks for reverting it already!